scripts/cicd/verify-gather-logs-operations.py

```diff
@@ -39,6 +39,15 @@
     'starter-us-west-2'
 ]
 
+# This regex matches a valid hostname: an optional set of DNS labels each
+# ending with dot, followed by a final DNS label. Each DNS label is 1 to
+# 63 digits, letters or '-', but '-' can't be the first or last character.
+HOSTNAME_RE = re.compile(
```

Comment:
Got it.  So long as we filter out malicious input, it's ok if we let an IP address in when we really wanted a hostname.  No problems here.